packages/flutter/lib/src/material/app.dart

```diff
@@ -132,9 +132,23 @@ class MaterialApp extends StatefulWidget {
        ),
        super(key: key);
 
-  /// A one-line description of this app for use in the window manager.
+  /// A one-line description used by the device to identify the app for the user.
+  ///
+  /// To provide a localized title instead, use [onGenerateTitle].
+  ///
+  /// This value is simply passed along to [WidgetsApp.title].
```

Comment:
rather than "simply passed along to", consider "passed unmodified to", to avoid claiming we are doing anything "simply".

Reply:
RIght! :-)